src/check_jsonschema/catalog.py

```diff
@@ -212,6 +212,16 @@ def _githubusercontent_url(owner: str, repo: str, ref: str, path: str) -> str:
             ],
         },
     },
+    "snapcraft": {
+        "url": _githubusercontent_url(
+            "canonical", "snapcraft", "main", "schema/snapcraft.json"
+        ),
+        "hook_config": {
+            "name": "Validate snapcraft files",
```

Comment:
The default description is a bit inaccurate in this case (see the lines above where it says the schema from SchemaStore).

```suggestion
            "name": "Validate snapcraft files",
            "description": (
                "Validate snapcraft files against the schema provided by Canonical"
            ),
```